fog/ledger/enclave/src/lib.rs

```diff
@@ -35,7 +35,9 @@ pub const ENCLAVE_FILE: &str = "libledger-enclave.signed.so";
 pub struct LedgerSgxEnclave {
     eid: sgx_enclave_id_t,
     // Hold a reference counter to the enclave to prevent destruction.
+    #[allow(dead_code)]
     enclave: Arc<SgxEnclave>,
+    #[allow(dead_code)]
     logger: Logger,
```

Comment:
Underscores should eliminate the dead code

```suggestion
    #[allow(dead_code)]
    _enclave: Arc<SgxEnclave>,
    #[allow(dead_code)]
    _logger: Logger,
```